public/static/docs/command-reference/repro.md

```diff
@@ -45,6 +45,44 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Running other dvc commands in parallel
+
+See
+[Running other dvc commands in parallel](/doc/command-reference/run#running-other-dvc-commands-in-parallel).
+
+### Parallel stage execution
```

Comment:
this is a good one! I'll @jorgeorpinel review specific language nuances, but I hope it will be useful for people. (When we at last have a How to section it can be probably moved there with a link to it from this document - just to keep a sane size)

Reply:
How to section?

Reply:
"How to" (name can change) - we have a bunch of tickets already that start like "write a How to **** "

Reply:
I see. OK, I created #899 to determine this.